libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -302,6 +302,63 @@ impl<'a> JsiiImporter<'a> {
 		self.register_jsii_type(&enum_fqn, &enum_symbol, enum_type_ref);
 	}
 
+	/// Import a JSII interface as a function instead.
+	/// These interfaces must have the @callable annotation and only one method defined, which will be the function signature.
+	fn import_closure(&mut self, jsii_interface: &wingii::jsii::InterfaceType) {
```

Comment:
@MarkMcCulloh just a question to understand in depth this change; would this new method allow closures coming from the `.jsii` compiled via the SDK? (as long as they have the `@typeparam` annotation in place)

Reply:
(assuming it was called from `import_interface`)

Reply:
If you create an interface with `@callable`, it will be imported as a function type instead of an interface:

```ts
/*
 * @callable
 */
export interface IFunkyFunction {
  fn: (arg: string) => number;
}
```

will become the wing function type:

```wing
(str): num
```

It can also use `@typeparam` to have generic args/return type